data/json/recipes/recipe_weapon.json

```diff
@@ -578,6 +605,17 @@
     "autolearn": true,
     "components": [ [ [ "bowling_pin", 1 ] ], [ [ "spike", 2 ] ], [ [ "duct_tape", 100 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "plunger_weapon",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "time": "30 s",
+    "reversible": true,
```

Comment:
Driving a spike through the handle won't be reversible 🤔

Reply:
recipe removed